package.json

```diff
@@ -11,7 +11,7 @@
 		"start": "cross-env NODE_ENV=production node --max-old-space-size=2048 server/index.js",
 		"start-k8s": "cross-env NODE_ENV=production node --max-old-space-size=1536 server/index.js",
 		"docker-dev-vm": "rimraf node_modules/.cache/hard-source && cross-env NODE_ENV=development node server/dev-server.js",
-		"inspect-dev": "cross-env NODE_ENV=development node --inspect --max-old-space-size=4096 server/dev-server.js",
+		"inspect-dev": "cross-env NODE_ENV=development TAILWIND_MODE=watch node --inspect --max-old-space-size=4096 server/dev-server.js",
```

Comment:
Out of curiousity when are we using this npm script?

Reply:
I typically use this for server side debugging and can add some info to the readme for setting up.